src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts

```diff
@@ -127,7 +131,12 @@ function areNeighborsOverlapping(neighbors: any[]) {
 function getStackLabelPointsForDualAxis(stacks: any[]) {
```

Comment:
please consider add unit test for this one.

Reply:
added new test